apps/wallet/src/ui/app/shared/indented-title/index.tsx

```diff
@@ -1,24 +1,39 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { Text } from '_app/shared/text';
+import { Disclosure } from '@headlessui/react';
+import { ChevronDown12, ChevronRight12 } from '@mysten/icons';
 
 import type { ReactNode } from 'react';
 
 type IndendedTitleProps = {
     title: string;
     children: ReactNode;
+    defaultOpen?: boolean;
 };
 
-export function IndentedTitle({ title, children }: IndendedTitleProps) {
+export function IndentedTitle({
```

Comment:
as a side note, I'm confused about this component's name

Reply:
Yeah I'll actually change this because I was confused too.